recipes/lora_finetune_single_device.py

```diff
@@ -96,6 +96,7 @@ def __init__(self, cfg: DictConfig) -> None:
         self.total_training_steps = 0
 
         self._resume_from_checkpoint = cfg.resume_from_checkpoint
+        self._gradient_accumulation_steps = cfg.gradient_accumulation_steps
```

Comment:
Thanks for adding this! Now only one recipe remains.. (though it's the one that actually needs this feature the least)